dev-vcs/qgit/qgit-2.9.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A Qt-based git GUI viewer"
+DESCRIPTION="With qgit you will be able to browse revisions history,\
+view patch content and changed files, graphically following different \
+development branches."
+HOMEPAGE="https://github.com/tibirna/qgit"
+COPYRIGHT="2011-2019 Cristian Tibirna"
```

Comment:
About window shows more:
```diff
- COPYRIGHT="2011-2019 Cristian Tibirna"
+ COPYRIGHT="2005-2008 Marco Costalba
+	2011-2019 Cristian Tibirna"
```

Reply:
Only now do I know that it's possible to include more than one line in `COPYRIGHT`.

Reply:
It's explained in the wiki guidelines :)

Reply:
Updated comment
Seems there were more people involved between 2005-2008